data/src/main/java/org/cryptomator/data/cloud/pcloud/RootPCloudFolder.java

```diff
@@ -8,10 +8,9 @@
 class RootPCloudFolder extends PCloudFolder {
 
 	private final PCloud cloud;
-	private static final long rootFolderId = RemoteFolder.ROOT_FOLDER_ID;
 
 	public RootPCloudFolder(PCloud cloud) {
-		super(null, "", "", rootFolderId);
+		super(null, "", "", (long) RemoteFolder.ROOT_FOLDER_ID);
```

Comment:
there is no need in Java to explicitly cast an int to long. invoking any method that accepts long with an int is completely safe

Reply:
The cast is needed since I need to provide a `Long`. `int` cannot be assigned to a `Long` if it's not casted to `long`.

Also changing the signature from `Long` to `long` isn't possible since it has to be nullable.

Reply:
> changing the signature from Long to long isn't possible since it has to be nullable.

I see. What a shame... 🙁